cinema/views.py

```diff
@@ -56,3 +84,49 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
+        date = self.request.query_params.get("date")
+        movie_id = self.request.query_params.get("movie")
+        if movie_id:
+            queryset = queryset.filter(movie__id=movie_id)
+        if date:
+            date = datetime.strptime(date, "%Y-%m-%d")
```

Comment:
The `datetime.strptime(date, "%Y-%m-%d")` line could raise a `ValueError` if the date format is incorrect. Consider adding a try-except block to handle this exception and provide a user-friendly error message.